doc/dev_guide/create_aws_sts_setup_on_minikube.md

```diff
@@ -0,0 +1,257 @@
+# Create AWS STS Setup On Minikube
+AWS Security Token Service (STS) is a web service that enables users to request temporary, limited-privilege credentials. In this guide, we would create a setup that eventually we will able to get those credentials (see the Test part). We would use OpenID Connection (OIDC) which is an identity layer that allows the verification of the identity.
+
+#### Requirements:
+- AWS account (with Admin permission)
+- AWS CLI
+- minikube
+- openssl
+
+### Background
+AWS STS allows applications to provide a Json Web Token (JWT) that can be used to assume an IAM role. The JWT includes an Amazon Resource Name (ARN) for `sts:AssumeRoleWithWebIdentity` IAM action to allow temporarily granted permission for the `ServiceAccount` to do this. The JWT contains the signing keys for the `ProjectedServiceAccountToken` that can be validated by AWS IAM.
+
+### Initial Setup - Create S3 Bucket (the OIDC Bucket)
+1. Create the AWS S3 bucket (for hosting the OIDC configurations). Please make it public access and ACL enabled after creation (see notes below).
+
+```bash
+aws s3api create-bucket --bucket <bucket_name> --region <aws_region> --create-bucket-configuration LocationConstraint=<aws_region>
+```
+
+#### Notes:
+- If the value of <aws_region> is us-east-1, do not specify the LocationConstraint parameter.
+- We will refer to this bucket name as <oidc_bucket_name> in the next instructions.
+- Change the publicity: in the AWS Console: block public access (bucket settings), do not block anything, and enable the bucket with ACL.
```

Comment:
This is a bit unclear. what do you mean by that? which checkboxes should be set?

Reply:
I changed the text and I'm adding the screenshots so it will be clear:
![Screenshot 2023-11-12 at 8 54 29](https://github.com/noobaa/noobaa-operator/assets/57721533/5725ebe6-24d2-4f8b-abaa-0971c57385fd)

![Screenshot 2023-11-12 at 8 54 19](https://github.com/noobaa/noobaa-operator/assets/57721533/f6ec2e95-2353-43b1-a770-0b2c997d46ad)